nss-day-cohort-48/rare-api-orchestra-outlaws
# SLM Post Tags

#### Changes Made

1. Added file `post_tag.py` to `models` directory.
2. Added file `post_tag.py` to `views` directory.
3. Added file `post_tags.json` to `fixtures` directory.
4. Modified file `rebuild-database` to include `post-tags`
5.  Modified file `urls.py` to include `post-tags viewset`

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout slm-posttags
   run ./rebuild-database
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.

   > Example: In Postman, run GET on /post_tags, /post_tags/{id}, run POST to create a new post_tag, run DELETE to destroy that post_tag. run GET /post_tags?post={id} as well as GET on /post_tags?tag={id} Ensure all returns as expected.
 
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
